### PR TITLE
Revisions for Ubuntu tutorial

### PR DESCRIPTION
**Pull request by asaunders**
I've made revisions based on your comments. Please let me know if there is anything else I can do to make this tutorial perfect, or if there is something that you feel I did not address properly in your critique.

Thanks!
Adam

### CODE REVIEW

File: tutorials/desktop/install-ubuntu-on-chromebook/install-ubuntu-on-chromebook.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Placing your device into Developer Mode will wipe all data and user information from it. Since ChromeOS is an online-centric operating system, the vast majority of your data will be stored remotely, but make sure you've backed up everything important that's local to your machine before you begin. This could include important data like any files you've downloaded, or locally cached passwords.
+The use of Developer Mode may void your Chromebook's warranty.
+
+To get to Developer Mode, we need to first reboot into Recovery Mode. On most Chromebooks, you do so by turning the device off, then holding down the `ESC` and `Refresh` keys while you press the Power button.
+
+Once in this mode, press `Ctrl-D`. You will be prompted with an opportunity to "turn OS verification OFF". Press Enter to do so.
+
+When you boot up your Chromebook, it will begin with a warning screen noting that "OS verification is OFF". You will need to press `Ctrl-D` to continue. Your device will now transition to Developer Mode.
+
+![]https://www.howtogeek.com/wp-content/uploads/2015/02/ximg_54f13e24a1e8f.jpg.pagespeed.gp+jp+jw+pj+ws+js+rj+rp+rw+ri+cp+md.ic.EsQ9doLxVp.jpg
```
Comment (evilnick) [inline tutorials/desktop/install-ubuntu-on-chromebook/install-ubuntu-on-chromebook.md]: While it is possible to link to external images (you need to enclose the link in `(` `)`, for various reasons we prefer to have local copies of the images. If there is a rights problem with this, it is probably better to drop the image.

File: tutorials/desktop/install-ubuntu-on-chromebook/install-ubuntu-on-chromebook.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+With the installation complete, you will be prompted to enter a username and a password. You will then be brought into a bare-bones Ubuntu setup. 
+
+Pressing `Ctrl-Alt-Shift-Back` and `Ctrl-Alt-Shift-Forward` will rotate you between ChromeOS and Ubuntu. Logging out of Ubuntu will drop you back into the ChromeOS terminal tab that ran the system.
+
+You can get back to Ubuntu in future sessions by typing `sudo startunity` into your bash shell.
+
+## Practical and fun uses for your system
+Duration: 3:00
+
+Like playing games? `sudo apt-get install steamv in a terminal gets you Valve's Steam gaming client for Linux-based desktop gaming on your Chromebook. If you're not a previous Steam user, you'll need to register an account.
```
Comment (caldav) [inline tutorials/desktop/install-ubuntu-on-chromebook/install-ubuntu-on-chromebook.md]: Please replace `sudo apt-get install` by `sudo apt install`.
And a typo: "steamv" -> "steam"

File: tutorials/desktop/install-ubuntu-on-chromebook/install-ubuntu-on-chromebook.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+Enjoy irony? `sudo apt-get install firefox`. Have fun running Firefox on ChromeOS!
```
Comment (caldav) [inline tutorials/desktop/install-ubuntu-on-chromebook/install-ubuntu-on-chromebook.md]: apt-get -> apt

File: tutorials/desktop/install-ubuntu-on-chromebook/install-ubuntu-on-chromebook.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+Want to browse for other ideas? `sudo apt-get install gnome-software ubuntu-software` gets you the Software tool for access to a wide variety of useful applications. Install LibreOffice for full office productivity, GIMP for desktop image editing, Audacity for audio editng, or Kodi for your multimedia enjoyment. If you're a developer, install your IDE of choice and hack away!
```
Comment (caldav) [inline tutorials/desktop/install-ubuntu-on-chromebook/install-ubuntu-on-chromebook.md]: apt-get -> apt

**Comment by asaunders** [comment]
Made fixes. Will email screenshot of crosh.

**Pull request merged**